tests/testMask.py

```diff
@@ -349,7 +349,7 @@ def testInterpret(self):
             self.assertEqual(im.interpret(bitmask), p)
             im.getArray()[0, i] = bitmask
             self.assertEqual(im.getAsString(i, 0), p)
-        self.assertEqual(self.Mask.interpret(allBits), ",".join(planes.keys()))
+        self.assertEqual(self.Mask.interpret(allBits), ",".join(sorted(planes.keys())))
```

Comment:
I think the conclusion from that discussion was that they never were supposed to be sorted to begin with. And that we should fix errors as we encounter them.